src/styles/unils/_variables.scss

```diff
@@ -0,0 +1,18 @@
+$cal-width: 100px;
+$fontSize: 30px;
```

Comment:
Firstly, use one kind of cases. Either camelCase or kebab one. Apply kebab
Secondly, this variable is not obvious. May skip it

Reply:
changed